librad/src/meta/project.rs

```diff
@@ -17,6 +21,75 @@ pub fn default_branch() -> String {
     DEFAULT_BRANCH.into()
 }
 
+#[derive(Debug, Fail)]
+pub enum Error {
+    #[fail(display = "Cannot serialize project metadata")]
```

Comment:
Would it not make sense to display the underlying error as well? I'm sure a user will ask "Why did the serialization fail?!"